### ISSUE TITLE
t2 update does not work over USB

### ISSUE THREAD

**Issue by ultralame** [tessel/t2-cli#318]
```
INFO Looking for your Tessel...
INFO Connected to Tessel-02A37FD59AC0 over USB
WARN Warning: Authorized LAN connection not detected for this Tessel. For a faster update, cancel and connect Tessel to LAN with the `tessel wifi` command before update.
events.js:85
      throw er; // Unhandled 'error' event
            ^
Error: Error upon remote process exit: 1
    at USBProcess.<anonymous> (/usr/local/lib/node_modules/t2-cli/lib/usb/usb_process.js:47:26)
    at USBProcess.g (events.js:199:16)
    at USBProcess.emit (events.js:107:17)
    at Parser.<anonymous> (/usr/local/lib/node_modules/t2-cli/lib/usb/usb_daemon.js:117:14)
    at Parser.emit (events.js:107:17)
    at Parser._write (/usr/local/lib/node_modules/t2-cli/node_modules/usb-daemon-parser/index.js:152:16)
    at doWrite (_stream_writable.js:301:12)
    at writeOrBuffer (_stream_writable.js:288:5)
    at Parser.Writable.write (_stream_writable.js:217:11)
    at ondata (_stream_readable.js:540:20)
    at emit (events.js:107:17)
    at readableAddChunk (_stream_readable.js:163:16)
    at Readable.push (_stream_readable.js:126:10)
    at InEndpoint.emit (events.js:107:17)
    at Transfer.transferDone (/usr/local/lib/node_modules/t2-cli/node_modules/usb/usb.js:295:9)
```

resolved by running `t2 update --timeout 10`

**Comment by johnnyman727** [comment]
Note for myself: this may just be an issue that happens every time an update is attempted over USB. Need to attempt to recreate.

**Comment by johnnyman727** [comment]
As of #486, this should be totally fixable!

**Comment by johnnyman727** [comment]
Fixed as of `0.0.18`.
